### ISSUE TITLE
Gradio automatically finds pretrained models to use

### ISSUE THREAD

**Issue by holycowdude** [SWivid/F5-TTS#730]
### Checks

- [x] This template is only for feature request.
- [x] I have thoroughly reviewed the project documentation but couldn't find any relevant information that meets my needs.
- [x] I have searched for existing issues, including closed ones, and found not discussion yet.
- [x] I confirm that I am using English to submit this report in order to facilitate communication.

### 1. Is this request related to a challenge you're experiencing? Tell us your story.

Is it possible to include an auto search within Gradio to find and select finetuned or custom models you've created locally?

### 2. What is your suggested solution?

Search on startup to search the e2-f5-tts.git\app\ckpts folder for all models and make them available to choose in inference

### 4. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by holycowdude** [comment]
Sounds Amazing! please share 😊

**Comment by dpp-user** [comment]
> I have something like that. I could post it when I get home next week if there's any interest.
> 
> * Dropdown for all models in /ckpts/ (with optional blacklist)
> * Dropdown for reference audio in /input/ (if a text file of the same name exists, it will populate the reference text field)
> * Toggle to save audio to /output/
> * Multi-speech toggle that normalizes each individual generation, rather than just the whole thing.
> * Multi-speech individual speed control.
> * GGUF chat model dropdown that you can load/unload at will and carry on the same conversation.
> 
> I've tried two GGUF models so far, Llama-3.2-3B-Instruct-Q5_K_M & Tiger-Gemma-9B-v3-Q4_K_M. They work great.
> 
> Would also like to try and add a multi-speech dropdown that allows you to use a different model for different speakers/types.
> 
> Edit: Haven't forget about this, give me a couple days!

can you share it? there are no updates to this software

**Comment by phudtran** [comment]
I've added support for F5TTS in [Jellybox](https://jellybox.com)

Includes 

- A model selector (Only GPT-SoVITS and F5TTS for now)
- Save your voices and generated audio to templates.
- Model downloader (Hosted on Huggingface)

Plan on adding voice chat with language models soon.

<img width="1013" alt="Image" src="https://github.com/user-attachments/assets/805d58f4-5475-4417-ae43-3903b8f0b3af" />

<img width="1023" alt="Image" src="https://github.com/user-attachments/assets/6a8cf41d-4cc9-46f8-a889-396ea4d27a6f" />

<img width="1026" alt="Image" src="https://github.com/user-attachments/assets/aac0a872-56d0-4467-a842-f09639f2def8" />

**Comment by kostianu4** [comment]
@phudtran is it possible to use this model, for ex. https://huggingface.co/hotstone228/F5-TTS-Russian/tree/main

**Comment by phudtran** [comment]
> [@phudtran](https://github.com/phudtran) is it possible to use this model, for ex. https://huggingface.co/hotstone228/F5-TTS-Russian/tree/main

Yeah it's possible. Would just need to convert it to onnx with https://github.com/DakeQQ/F5-TTS-ONNX

**Comment by SWivid** [comment]
Hey guys, feel free to check if v1.1.3 meet your need~

@Jxspa always welcome if willing to improve chat feature stuff
